src/citrine/resources/predictor_evaluation_execution.py

```diff
@@ -27,7 +27,7 @@ class PredictorEvaluationExecution(Resource['PredictorEvaluationExecution']):
 
     _response_key = None
 
-    uid = properties.UUID('id', serializable=False)
+    uid: UUID = properties.UUID('id', serializable=False)
     """UUID of the execution."""
```

Comment:
PyCharm seems to incorrectly infer the type of this variable as a string. (I think because of the docstring.)